charmcraft.yaml

```diff
@@ -33,6 +33,17 @@ parts:
     override-build: |
       rustup default stable
       craftctl default
+  app:
+    plugin: nil
+    override-build: |
+      cd /root/project
+      tar -cvzf app.tar.gz app
```

Comment:
needed to create a tar, because local install using pipx lead to permission errors (pipx is creating an `egg-info` directory without permissions